index.js

```diff
@@ -511,6 +522,20 @@ module.exports = {
     return true;
   },
 
+  _emberDataVersionRequiresPackagesPolyfill() {
+    let checker = new VersionChecker(this.project);
+    let dep = checker.for('ember-data');
+    let hasEmberData = dep.exists();
+
+    if (hasEmberData) {
+      if (!dep.version) {
+        throw new Error('EmberData missing version');
+      }
```

Comment:
Do we ever expect this to actually happen? I _think_ no, but just wanted to check and see why it was added...

Reply:
When I was creating the test mock environment the failures were pretty obtuse, this made it clear when the issue was no version found so that I knew I needed to continue getting the mock env closer to what version checker needed.

It is removable.